pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +391,78 @@ class OSRow extends React.Component {
+    const validationStatePassword = validationFailed.password ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+        </label>
+    );
+    if (unattendedDisabled) {
+        unattendedInstallationCheckbox = (
+            <OverlayTrigger overlay={ <Tooltip id='os-unattended-installation-tooltip'>{ _("The selected Operating System does not support unattended installations") }</Tooltip> } placement='left'>
+                {unattendedInstallationCheckbox}
+            </OverlayTrigger>
+        );
+    }
+
+    return (
+        <>
+            {unattendedInstallationCheckbox}
+            {!unattendedDisabled && unattendedInstallation ? <>
+                {os.profiles.length > 0 && <>
+                    <label className="control-label" htmlFor="profile-select">
+                        {_("Profile")}
+                    </label>
+                    <Select.Select id="profile-select"
+                        initial={os.profiles && os.profiles[0]}
+                        onChange={e => onValueChanged('profile', e)}>
+                        { (os.profiles || []).sort()
+                                .reverse() // Let jeos (Server) appear always first on the list since in osinfo-db it's not consistent
+                                .map(profile => {
+                                    let profileName;
+                                    if (profile == 'jeos')
+                                        profileName = 'Server';
+                                    else if (profile == 'desktop')
+                                        profileName = 'Workstation';
+                                    else
+                                        profileName = profile;
+                                    return <Select.SelectEntry data={profile} key={profile}>{profileName}</Select.SelectEntry>;
+                                }) }
+                    </Select.Select>
+                </>}
+                <label htmlFor='root-password' className='control-label'>
+                    {_("Root Password")}
+                </label>
+                <FormGroup validationState={validationStatePassword} bsClass='form-group ct-validation-wrapper' controlId='root-password'>
+                    <Password id='root-password' onValueChanged={(value) => onValueChanged('rootPassword', value)} />
+                    <HelpBlock>
+                        <p className="text-danger">{validationFailed.password}</p>
+                        <p className="text-info">{_("Leave the password blank if you do not wish to have a root account created")}</p>
```

Comment:
Is that actually true? When trying to not set a password, I get:

    (red error) Please set a root or a user Password
    (blue hint) Leave the password blank if you do not wish to have a root account created

So perhaps the info shouldn't be shown when unattended is checked and there's no user account input line.